javascript/node/selenium-webdriver/test/devtools_test.js

```diff
@@ -35,15 +35,18 @@ suite(
     after(async () => await driver.quit())
 
     ignore(browsers(Browser.CHROME)).it(
-      'sends Page.enable command using devtools', async function () {
+      'sends Page.enable command using devtools',
+      async function () {
         const cdpConnection = await driver.createCDPConnection('page')
         cdpConnection.execute('Page.enable', {}, function (_res, err) {
           assert(!err)
         })
-      })
+      }
```

Comment:
HI @potapovDim,

Changes looks good!

But in few places I see this is moved to multi-line

`})` is more readable I think!

Reply:
@harsha509 those changes were done based on prettier and eslint codestyle formats